lib/rules/no-curly-component-invocation.js

```diff
@@ -47,14 +49,29 @@ const BUILT_INS = new Set([
 
 const ALWAYS_CURLY = new Set(['yield']);
 
+function toAngleAttrValue(attr) {
+  const node = attr.value;
+  if (node.type === 'SubExpression') {
+    return b.mustache(node.path, node.params, node.hash);
+  }
+  if (node.type === 'StringLiteral') {
+    if (node.original === '') {
+      return b.mustache('concat', [b.string('')]);
+    }
```

Comment:
Whats this for? I don't think we would normally want to emit a concat statement here (I would expect to do the `b.text(node.original)` just below) 🤔

Reply:
@rwjblue  it not working for `empty` string

Reply:
@rwjblue https://astexplorer.net/#/gist/e7b5fc128add06ab02974faea2a979cd/b7652f5886bfb214f84bab0721d3b4148fd4d42d

Reply:
@rwjblue fixed by toggling `isValueLess`